### PR TITLE
Use a newer screenshot image viewer

### PR DESCRIPTION
**Pull request by VSEphpbb**
Also fixes https://tracker.phpbb.com/projects/CUSTDB/issues/CUSTDB-730

### CODE REVIEW

File: styles/prosilver/template/contributions/contribution_header.html [hunk 1]
```diff
@@ -1,16 +1,5 @@
 <!-- INCLUDE overall_header.html -->
 
-<!-- IF U_DOWNLOAD and U_SUBSCRIBE and not IS_SUBSCRIBED and not QUEUE_STATUS -->
```
Comment (DavidIQ) [inline styles/prosilver/template/contributions/contribution_header.html]: Why was this removed?

File: styles/prosilver/theme/colorbox.css [hunk 2]
```diff
@@ -1,70 +0,0 @@
-/*
```
Comment (DavidIQ) [inline styles/prosilver/theme/colorbox.css]: I assume this file was just deleted and github's GUI is being dumb and simply showing all lines were removed?

**Comment by DavidIQ** [comment]
If we're no longer needing colorbox then maybe `assets/javascript/jquery.colorbox-min.js` should also be removed and the reference to it removed from `styles/all/template/event/overall_footer_after.html`?

**Comment by VSEphpbb** [comment]
@DavidIQ `assets/javascript/jquery.colorbox-min.js` is being removed along with any references to it.

File: styles/prosilver/theme/colorbox.css (hunk 2 above)
Comment (VSEphpbb) [inline styles/prosilver/theme/colorbox.css]: yep

File: styles/prosilver/template/contributions/contribution_header.html (hunk 1 above)
Comment (VSEphpbb) [inline styles/prosilver/template/contributions/contribution_header.html]: Because this is essentially dead code. From what I have tried to infer, this appeared to be part of an option to Subscribe to a contribution when you have chosen to download one. There is no more code in Titania that unhides this...nor I have I ever remembered experiencing such a thing.

**Comment by VSEphpbb** [comment]
FYI the plugin can be demo'ed here
https://noelboss.github.io/featherlight/gallery.html

**Comment by DavidIQ** [comment]
Yeah, magnific is one I use in ReIMG so... 👍
